README.md

```diff
@@ -24,3 +25,4 @@ The [definitions and clarifications](definitions.md) for terms used in these pol
 | [Policy 6](policies/accepted/6.md) | *passed* | 11 | 0 | 3 | 2 | 16 | 1st of August 2020
 | [Policy 7](policies/accepted/7.md) | *passed* | 11 | 0 | 3 | 2 | 16 | 1st of August 2020
 | [Policy 8](policies/accepted/8.md) | *passed* | 7 | 0 | 1 | 0 | 8 | 2nd of August 2020
+| [Policy 9](policies/accepted/9.md) | *passed* | 8 | 0 | 0 | 0 | 8 | 13th of October 2020
```

Comment:
```suggestion
| [Policy 9](policies/accepted/9.md) | *passed* | 8 | 0 | 0 | 0 | 8 | 5th of October 2020
```

Tiny nitpick here, the vote itself was called on the 3th and closed on the 5th as per policy 7.